run/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+file(GLOB_RECURSE data_src "data/*")
+
+if(WIN32)
+    # Ensure that r2pkg can find its dependent DLLs
+    set(SETENV_FOR_UPDATE_DATA
+            ${CMAKE_COMMAND} -E env
+            "PATH=${REAPING2_DEPS_INSTALL_DIR}/bin$<SEMICOLON>$<TARGET_FILE_DIR:r2pkg>")
+endif(WIN32)
+
+add_custom_command(OUTPUT data.pkg
+        COMMAND
+            ${SETENV_FOR_UPDATE_DATA}
+            r2pkg -c ${CMAKE_CURRENT_SOURCE_DIR}/data
```

Comment:
Ok, so which would you prefer:

1. When crosscompiling, don't add update_data to ALL
2. When crosscompiling, use find_program to check if r2pkg is available (and if not, skip the update target?)